tools/osbuilder/rootfs-builder/ubuntu/rootfs_lib.sh

```diff
@@ -75,4 +75,10 @@ build_rootfs() {
     # Reduce image size and memory footprint
     # removing not needed files and directories.
     chroot $ROOTFS_DIR rm -rf /usr/share/{bash-completion,bug,doc,info,lintian,locale,man,menu,misc,pixmaps,terminfo,zoneinfo,zsh}
+
+ 	if [ "${AA_KBC}" == "eaa_kbc" ] && [ "${ARCH}" == "x86_64" ]; then
+		echo 'deb [arch=amd64] http://mirrors.openanolis.cn/inclavare-containers/ubuntu20.04 bionic main' | chroot $ROOTFS_DIR tee /etc/apt/sources.list.d/inclavare-containers.list
+		wget -qO - http://mirrors.openanolis.cn/inclavare-containers/ubuntu20.04/DEB-GPG-KEY.key  | chroot $ROOTFS_DIR apt-key add -
+		chroot $ROOTFS_DIR apt-get update && chroot $ROOTFS_DIR apt-get install -y rats-tls
```

Comment:
> I tried command "chroot $ROOTFS_DIR apt-get update && apt-get install -y rats-tls" and found it unworkable.

@liangzhou121 because that will run the `install` outside the chroot :) what about
```suggestion
	if [ "${AA_KBC}" == "eaa_kbc" ]; then
		wget -qO - http://mirrors.openanolis.cn/inclavare-containers/ubuntu20.04/DEB-GPG-KEY.key | chroot $ROOTFS_DIR apt-key add -
		cat << EOF | chroot $ROOTFS_DIR
echo 'deb [arch=amd64] http://mirrors.openanolis.cn/inclavare-containers/ubuntu20.04 bionic main' > \
	/etc/apt/sources.list.d/inclavare-containers.list
apt-get update
apt-get install -y rats-tls
EOF
```

Reply:
Thank you. I updated it according to your suggestion.